README.md

```diff
@@ -1 +1,193 @@
-# Hello-Microverse-1
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+    <h3><b>Hello-Microverse</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
```

Comment:
- [ OPTIONAL ] I noticed that the **Future Features** section was excluded from your README file. Please note that this section is compulsory to fill when setting up your Readme file and in future projects, if you leave it out, your project will be invalidated. So I would advise that you add it before merging this branch and make sure to include it in all future projects. :wink: